crates/rpc/rpc-builder/src/constants.rs

```diff
@@ -20,7 +22,9 @@ pub const DEFAULT_MAX_BLOCKS_PER_FILTER: u64 = 100_000;
 pub const DEFAULT_MAX_LOGS_PER_RESPONSE: usize = 20_000;
 
 /// The default maximum number of concurrently executed tracing calls
-pub const DEFAULT_MAX_TRACING_REQUESTS: u32 = 25;
+pub static DEFAULT_MAX_TRACING_REQUESTS: Lazy<usize> = Lazy::new(|| {
+    std::thread::available_parallelism().map_or(25, |x| max(usize::from(x).saturating_sub(2), 2))
+});
```

Comment:
this should be a function

Reply:
what do you mean? Should I extract the whole lambda function which passed to `Lazy::new` into a named function?

Reply:
yes

Reply:
Done